tests/posit/connect/test_content.py

```diff
@@ -165,6 +187,30 @@ def test_tags(self):
         assert self.item.tags == []
 
 
+class TestContentItemGetContentOwner:
+    @responses.activate
+    def test_owner(self):
+        mock_content = load_mock(
+            "v1/content/f2f37341-e21d-3d80-c698-a935ad614066.json"
+        )
+        responses.get(
+            "https://connect.example/__api__/v1/content/f2f37341-e21d-3d80-c698-a935ad614066",
+            json=mock_content,
+        )
+
+        responses.get(
+            f"https://connect.example/__api__/v1/users/20a79ce3-6e87-4522-9faf-be24228800a4",
+            json=load_mock(
+                f"v1/users/20a79ce3-6e87-4522-9faf-be24228800a4.json"
+            ),
+        )
+
+        c = Client("12345", "https://connect.example")
+        item = c.content.get("f2f37341-e21d-3d80-c698-a935ad614066")
+        owner = item.owner
+        assert owner.guid == "20a79ce3-6e87-4522-9faf-be24228800a4"
```

Comment:
Other things to assert: 

* owner is ContentItemOwner (if we care about that)
* The user GET on L201 was called once
* Do item.owner again and confirm that L201 has still only been called once.